designs/administrator/future-openshift/network-policy-form/index.md

```diff
@@ -0,0 +1,61 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Network policy form
+
+## Background
+Setting Network policies is a critical part of the security of applications on a cluster. In OCP 4.7 the only way to manage Network policy in the console is to edit the yaml of these resources. It's a common and important enough operation that we should make this easier for administrators to work with.
+
+## Goal
+
+Add the ability to create/update/delete NetworkPolicy resources through the OpenShift console in a way that is easier than directly editing the yaml representation of these resources.
+
+## Designs for 4.9 release
+
+This initial concept is a form based solution that compliments the current YAML editor. This allows users to switch between the form and YAML views so they may better understand the YAML output.
+
+### Create Network Policy introduction
+
+* The policy will pre-fill the namespace based on the Project selector at the top.
```

Comment:
Does this mean the namespace would still be editable in the form? This is something we generally struggle with consistency-wise in the console (see issue https://github.com/openshift/openshift-origin-design/issues/507.) I was wondering if changing the namespace in the field would have any affect on the project selector's value.

Reply:
This will be removed from the form to remove that potential confusion. Thanks for the input!